apidoc/Titanium/Network/TCPSocket.yml

```diff
@@ -33,8 +33,10 @@ events:
     properties:
       - name: from
         summary: the reference for the socket that data was retrieved from
+        type: Number
```

Comment:
Is it really `Number` or am I missing something (I have no experience in obj-c)?

Reply:
yes, it is.